daemon/container_operations_unix.go

```diff
@@ -63,7 +63,11 @@ func (daemon *Daemon) getIpcContainer(container *container.Container) (*containe
 	if err != nil {
```

Comment:
We need access to supplied container's hostconfig (to call its IpcMode.Container() method), I can't see how this can be done with the change you propose.